docs/developer/architecture/communication_flows.rst

```diff
@@ -0,0 +1,213 @@
+.. _communication_flows:
+
+Communication Flows
+===================
+
+This section presents communication sequences organized by WebMon functionality.
+
+.. contents:: :local:
+
+Experiment monitoring
+---------------------
+
+Instrument status and PV updates
+................................
+
+DASMON, from Data Acquisition (DAQ) System Monitor, provides instrument status and process variable
+(PV) updates from the beamlines to WebMon. DASMON connects to the WebMon message broker to pass
+status information, for example the current run number and count rate, to Dasmon listener. Due to
+the high volume of PV updates, DASMON writes PV:s directly to the PostgreSQL database.
+
+.. mermaid::
+
+    sequenceDiagram
+        participant DASMON
+        participant Dasmon listener
+        participant Workflow DB
+        par
+            DASMON->>Workflow DB: PV update
+        and
+            DASMON->>Dasmon listener: Instrument status
+            Dasmon listener->>Workflow DB: Instrument status
+        end
+
+Run status updates
+..................
+
+The Stream Management Service (SMS) posts messages on the queue ``APP.SMS`` at run start, run stop
+and when the Streaming Translation Client (STC) completes translation to NeXus.
+
+.. mermaid::
+
+    sequenceDiagram
+        participant SMS
+        participant Dasmon listener
+        participant Workflow DB
+        SMS->>Dasmon listener: Run started
+        Dasmon listener->>Workflow DB: Add new data run
+        Dasmon listener->>Workflow DB: Run status
+        SMS->>Dasmon listener: Run stopped
+        Dasmon listener->>Workflow DB: Run status
+        SMS->>Dasmon listener: Translation succeeded
+        Dasmon listener->>Workflow DB: Run status
+
+
+Experiment data post-processing
+-------------------------------
+
+Autoreduction and cataloging
+............................
+
+The sequence diagram below describes the communication flow as a run gets post-processed.
+The post-processing workflow is triggered when the Streaming Translation Client (STC) has finished
+translating the data stream to NeXus and sends a message on the queue ``POSTPROCESS.DATA_READY``
+specifying the instrument, IPTS, run number and location of the NeXus file.
+
+The post-processing workflow for the instrument is configurable in the database table
+``report_task``.
+The diagram shows the three post-processing steps that are available: autoreduction, cataloging of
+raw data in `ONCat <https://oncat.ornl.gov/>`_ and cataloging of reduced data in
+`ONCat <https://oncat.ornl.gov/>`_.
+
```

Comment:
I thought there was going to be a “note” stating that while what is represented in the diagram the sequence of the workflow is configured for different instruments and may be in a different order